Makefile

```diff
@@ -2,7 +2,7 @@
 
 PACKAGES_SIMTEST=$(shell go list ./... | grep '/simulation')
 # VERSION := $(shell echo $(shell git describe --tags) | sed 's/^v//')
-VERSION := "0.0.5"
```

Comment:
we should probably hold on the patch until some other chores are performed. At which point we should make a PR with changelog updates.

Reply:
Done